tests/framework/utils/defaults.sh.in

```diff
@@ -17,11 +17,7 @@ set_distro_vars() {
 
     if [ "$distro" = "el7" ]; then
         if ${upgrade_test}; then
-            if ${UPGRADE_FROM_3:-false}; then
-                export TEST_DISTRO_VERSION=${TEST_DISTRO_VERSION:-"7.3"}
-            else
-                export TEST_DISTRO_VERSION=${TEST_DISTRO_VERSION:-"7.4"}
-            fi
+            export TEST_DISTRO_VERSION=${TEST_DISTRO_VERSION:-"7.5"}
```

Comment:
Why not 7.6?

Reply:
This is the the *starting* distro. We go from IML 4 + EL 7.5, to IML 5 + EL 7.6

Reply:
ok that makes sense.